test/http/loopback.go

```diff
@@ -110,44 +110,3 @@ func ClientLoopback(t *testing.T, tc TapTest, topts ...cehttp.Option) {
 		assertTappedEquality(t, "http response", tc.asRecv, &resp)
 	}
 }
-
-// To help with debug, if needed.
-func printTap(t *testing.T, tap *tapHandler, testID string) {
```

Comment:
please leave this, it was really helpful for debugging.

Reply:
ok